tests/pingpong.rs

```diff
@@ -13,29 +11,37 @@ use self::common::Test;
 use ::demikernel::{
     Ipv4Endpoint,
     OperationResult,
+    QDesc,
+    QToken,
 };
 use ::std::{
     panic,
     process,
-    sync::mpsc,
+    sync::{
+        mpsc,
+        mpsc::{
+            Receiver,
+            Sender,
+        },
+    },
     thread,
+    thread::JoinHandle,
     time::Duration,
 };
 
 //==============================================================================
-// Ping Pong
+// UDP Ping Pong
 //==============================================================================
 
 #[test]
 fn udp_ping_pong() {
-    let mut test = Test::new();
-    let mut npongs: usize = 1000;
-    let payload: u8 = 'a' as u8;
+    let mut test: Test = Test::new();
+    let fill_char: u8 = 'a' as u8;
     let local_addr: Ipv4Endpoint = test.local_addr();
     let remote_addr: Ipv4Endpoint = test.remote_addr();
 
     // Setup peer.
-    let sockfd = test
+    let sockfd: QDesc = test
         .libos
         .socket(libc::AF_INET, libc::SOCK_DGRAM, 0)
         .unwrap();
```

Comment:
Still 4 of these left.

Reply:
Thanks for fixing the others though, looks much better now.

Reply:
Oh, sure... I didn't meant to touch the UDP testing code in this PR, that's why I haven't fixed. I'll do it.

Reply:
Thanks for fixing these.